### PR TITLE
Update Docs to Match with Other Asgardeo SPA SDKs

### PR DESCRIPTION
**Pull request by JanithGan**
## Purpose

$subject

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,14 +133,14 @@ npm install && npm start
 
 ### 1. Installing the Package
 
-Install the Angular library from the npm registry.
+Install the Asgardeo auth angular library from the npm registry.
```
Comment (thivi) [inline README.md:136]:
```suggestion
Install the Asgardeo Auth Angular SDK from the npm registry.
```

File: README.md [hunk 2]
```diff
@@ -131,14 +133,14 @@ npm install && npm start
...
 ```bash
 npm install --save @asgardeo/auth-angular
 ```
 
 ### 2. Import `AsgardeoAuthModule` and Provide Configuration Parameters
 
-Add the `AsgardeoAuthModule` to the imports array of your root app module. Pass the config object to the `forRoot` function. See all the available configurations list [here](#configuration).
+Add the `AsgardeoAuthModule` to the imports array of your root app module. Pass the config object to the `forRoot` function. See list of available configurations [here](#configuration).
```
Comment (thivi) [inline README.md:143]:
```suggestion
Add the `AsgardeoAuthModule` to the imports array of your root app module. Pass the config object to the `forRoot` function. See the list of available configurations [here](#configuration).
```

**Review by thivi — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (brionmario) [inline README.md:143]: Let's link Angular documentation regarding `forRoot`.

Comment (JanithGan) [reply README.md]: Fixed in ad37557

**Review by thivi — APPROVED** [review]

**Pull request merged**